token_utils/unpack.py

```diff
@@ -0,0 +1,47 @@
+"""
+Token unpacking and verifying functions.
+"""
+
+from time import time
+
+from django.conf import settings
+from jwkest import Expired, Invalid, MissingKey, jwk
+from jwkest.jws import JWS
+
+
+def unpack_jwt(token, lms_user_id, now=None):
+    """
+    Unpack and verify an encoded JWT.
+
+    Validate the user, expiration, and block.
+
+    Arguments:
+        token (string): The token to be unpacked and verified.
+        lms_user_id (int): LMS user ID this token should match with.
+        now (int): Optional now value for testing.
+    """
```

Comment:
this doctring should also say what the return value is (the payload obviously, but it would be nice to say what type the payload actually is)